routes/index.js

```diff
@@ -12,12 +12,15 @@ import permissionRoute from "./permission/index.js";
 
 import employeeRoute from "./employee/index.js";
 
+import leaveRequest from "./leaveRequest/index.js";
+
 const protectedRouter = express.Router();
 const unProtectedRouter = express.Router();
 
 // Protected Routes
```

Comment:
Why this comment?

Reply:
the comment is sir for the future incoming protected route